great_expectations/core/expectation_configuration.py

```diff
@@ -969,9 +969,6 @@ def __init__(
         self.meta = meta
         self.success_on_last_run = success_on_last_run
         self._ge_cloud_id = ge_cloud_id
-
-        if expectation_context is None:
-            expectation_context = ExpectationContext()
         self._expectation_context = expectation_context
```

Comment:
If you'd like to use the new functionality, you must explicitly pass it in. Is this a fair assumption?

Not sure if this violates any expected behavior that Cloud relies on.